onchain/permissionless-arbitration/coverage.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+forge coverage --report lcov --no-match-test Proofs
```

Comment:
We can remove the `--no-match-test Proofs` option, it's also moving to `matchine-solidity-step`